src/wakefs/fuse.cpp

```diff
@@ -61,6 +61,20 @@ bool json_as_struct(const std::string &json, json_args &result) {
 
   for (auto &x : jast.get("visible").children) result.visible.push_back(x.second.value);
 
+  JAST timeout_entry = jast.get("command_timeout");
+  if (timeout_entry.kind == JSON_INTEGER) {
+    int timeout = std::stoi(timeout_entry.value);
+    if (timeout <= 0) {
+      std::cerr << "timeout must be be an integer value greater than 0" << std::endl;
+      return false;
+    }
+
+    result.command_timeout = wcl::make_some<int>(timeout);
+  } else if (timeout_entry.kind != JSON_NULLVAL) {
```

Comment:
We probably don't care too much, but this does mean that `"isolate-pids": null` passes validation

Reply:
Yeah, the `get`+`kind` API doesn't distinguish between "key not set" and "key set to null"